src/core/TableRow/index.tsx

```diff
@@ -20,7 +20,7 @@ const TableRow = forwardRef<HTMLTableRowElement, RowProps>(
       tooltipProps,
       tooltipText = false,
       tooltipSubtitle,
-      hover,
+      hover = true,
```

Comment:
so the row tooltip will show itself unless it's explicitly told not to? that seems a little backwards from what I would expect...

Reply:
The `hover` prop determines if the row is hoverable or not (The background-color of a hoverable row changes on hover)
On the other hand, the condition to show a row tooltip is `if (shouldShowTooltipOnHover && hover)`. 
So we won't show the tooltip if the row is not hoverable.